packages/icons/package.json

```
@@ -0,0 +1,31 @@
+{
+  "name": "@sit-onyx/icons",
+  "description": "SVG icons for the onyx design system",
```

in the README.md logo, the phrasing is
```suggestion
  "description": "SVG icons for the onyx design systems",
```
should we align?

---

If we want to align this we should also do it for all other packages and GitHub documentation.
Imho for now we only have one design system so it would be fine for me